commit a4fbbb0227783c009c53da8f700f42ecc526880a
Author: Aws Ismail <aws.ismail@windriver.com>
Date:   Wed Sep 5 11:49:52 2012 -0400

    Configure SHA1 option properly
    
    Work around a cross-compiling configuration problem by
    defaulting the USE_SYSTEM_CRYPT_SHA1 option.
    
    This patch was migrated from WRL4.3
    
    Signed-off-by: Greff Moffatt <gregg.moffatt@windriver.com>
    
    Signed-off-by: Aws Ismail <aws.ismail@windriver.com>

diff --git a/configure.ac b/configure.ac
index fa6300f..3978c43 100644
--- a/configure.ac
+++ b/configure.ac
@@ -232,32 +232,6 @@ else
                      [return value of an overflowed snprintf])
 fi
 
-AC_MSG_CHECKING(whether you already have a standard SHA1 implementation)
-AC_RUN_IFELSE([AC_LANG_SOURCE([[
-#include <stdio.h>
-#include <string.h>
-#include <sys/types.h>
-#include <sha1.h>
-
-int main(void)
-{
-    SHA1_CTX ctx;
-    char b[41];
-    
-    SHA1Init(&ctx);
-    SHA1Update(&ctx, (const unsigned char *) "test", 4U);
-    SHA1End(&ctx, b);
-    b[40] = 0;
-    
-    return strcasecmp(b, "a94a8fe5ccb19ba61c4c0873d391e987982fbbd3");
-}
-]])],[
-AC_MSG_RESULT(yes)
-AC_DEFINE(USE_SYSTEM_CRYPT_SHA1,,[Define if you already have standard
-SHA1 functions])
-],[AC_MSG_RESULT(no)
-],[])
-
 AC_MSG_CHECKING([whether syslog names are available])
 AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
 #define SYSLOG_NAMES 1
